hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DataStorage.java

```diff
@@ -1071,12 +1071,26 @@ private static void linkAllBlocks(File fromDir, File fromBbwDir, File toDir,
   }
 
   private static class LinkArgs {
-    File src;
-    File dst;
+    private File srcDir;
+    private File dstDir;
+    private String blockFile;
+
+    LinkArgs(File srcDir, File dstDir, String blockFile) {
+      this.srcDir = srcDir;
+      this.dstDir = dstDir;
+      this.blockFile = blockFile;
+    }
+
+    public File src() {
```

Comment:
`getSrc` shoud be more graceful? The same as method `dst(...)` + `blockFile()`.

Reply:
it was meant to cause less code churn --> src ==> src() :)

Reply:
Yea I kept src() and dst() just to reduce the changes in the rest of the code and make it easier to review. Then I used blockFile rather than getBlockFile to keep it consistent.

I think its OK like this, as this is an inner class only used internally by DataStorage.

What do you think?